src/yocto/recipes.md

```diff
@@ -0,0 +1,186 @@
+| sharedstatedir	   |        /com           |
+| localstatedir	       |        /var           | 
+| datadir	           |        /usr/share     |
+| infodir	           |        /usr/info      |
+| mandir	           |        /usr/man       |
+| docdir	           |        /usr/doc       |
+| systemd_unitdir	   |        /usr/lib/systemd |
+| systemd_system_unitdir  | 	/usr/lib/systemd/ system |
+| systemd_user_unitdir	  |     /usr/lib/systemd/user |  
+| includedir	          |     /usr/include |  
+
+6. 8. do_package  
+6. 9. do_patch  
+6. 10. do_unpack  
+### 7. Priority of the recipe
+Variable name:  
+- PRIORITY =  
+Indicates the importance of a package, this depends on the purpose for which the distribution is being produced.
+You can set:
+- "required"
+- "standard"
+- "extra"
+- "optional"
+### 8. WORKDIR variable
+It is the pathname of the work directory in which the OpenEmbedded build system builds a recipe.  
+The WORKDIR directory is defined as:
+``` ${TMPDIR}/work/${MULTIMACH_TARGET_SYS}/${PN}/${EXTENDPE}${PV}-${PR} ``` 
+
+## Example for application recipe 
+For recipe file with the name:  helloworld.bb 
+```
+    DESCRIPTION = "Program that print Hello World! to standar oputput. 
+    PRIORITY = "optional" 
+    SECTION = "Examples" 
+    LICENSE = "MIT" 
+    LIC_FILES_CHKSUM = "file://${COMMON_LICENCE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302" 
+    SRC_URI = "file://helloworld.c" 
+    S = "${WORKDIR}" 
+    do_compile() { 
+    ${CC} ${CFLAGS} ${LDFLAGS} helloworld.c -o helloworld 
+    } 
+    do_install() { 
+    install -d ${D}${bindir} 
+    install -m 0755 helloworld ${D}${bindir} 
+    } 
+```
+# .bbappend file
+Recipes used to append Metadata to other recipes, there are calls append files, these files use the .bbappend file type. This file is used to modify the recipe file, this should have the same name that the recipe, at the same time bbappend files allow your layer to make additions or changes to the content of another layer's recipe without having to copy the other recipe into your layer.  
+Being able to append information to an existing recipe not only avoid duplication, also automatically applies recipe changes in a different layer to your layer.
+## Extending recipes with .bbappend files
+It is not necessary to recreate entire recipe files from scratch,  you can use **. bbappend** files to supplement an existing recipe file with new information, providing that the original information in the recipe file, resides in an existing layer.
+## Systemd recipes
+To use systemd services has necessary enabled this in the final image, because this is not enabled by default. Add the next lines to the configuration file **local. conf** to enable system as a default unit manager.
+
+```
+DISTRO_FEATURES_append = " systemd"
+DISTRO_FEATURES_BACKFILL_CONSIDERED += "sysvinit"
+VIRTUAL-RUNTIME_init_manager = "systemd"
+VIRTUAL-RUNTIME_initscripts = "systemd-compat-units"
+```
+
+### Important varibales for systemd recipe
```

Comment:
this section is too cryptic, try to be more direct and clear with your explanation